components/dialog/dialog-mixin.js

```diff
@@ -41,7 +41,10 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 			 * Whether or not the dialog is open
 			 */
 			opened: { type: Boolean, reflect: true },
-
+			/**
+			 * Opt out of dialog content scrolling
+			 */
+			noContentScroll: { type: Boolean, attribute: 'no-content-scroll', reflect: true },
```

Comment:
Maybe could be an ADVANCED property for the Daylight docs?

Reply:
Yeah that's a good call. Will add that.